pkg/volume/csi/csi_util.go

```diff
@@ -140,3 +143,23 @@ func hasReadWriteOnce(modes []api.PersistentVolumeAccessMode) bool {
 	}
 	return false
 }
+
+func isUnimplementedMethodErr(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	status, ok := status.FromError(err)
+	if !ok {
+		return false
+	}
+
+	methodIsNotImplemented :=
+		status.Code() == codes.Unimplemented && strings.Contains(status.Message(), "unknown method")
```

Comment:
I don't think the "unknown method" check is correct here since I think the message could be anything. Just checking for the `Unimplemented` code is better